yaus_c4gt_frontend/src/pages/SignIn.js

```diff
@@ -0,0 +1,323 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { useHistory } from "react-router-dom";
+import {
+  Layout,
+  Menu,
+  Button,
+  Row,
+  Col,
+  Typography,
+  Form,
+  Input,
+  Switch,
+} from "antd";
+import signinbg from "../assets/images/img-signin.jpg";
```

Comment:
Make it the relative paths instead of `../assets/images/img-signin.jpg`, it should be directly `assets/images/img-signin.jpg`